openfga/v1/openfga.proto

```diff
@@ -32,6 +32,57 @@ message Object {
   ];
 }
 
+// User represents any possible value for a user (subject)
+//
+// Can either be a:
+// - Specific user object (ex: 'user:will')
+// - Specific userset (ex: 'group:engineering#member')
+// - Public-typed wildcard (ex: 'user:*')
+//
+// See https://docs.fga.dev/intro/okta-fga-concepts#what-is-a-user
```

Comment:
Do not link to Okta docs please.

Use:
```suggestion
// See https://openfga.dev/docs/concepts#what-is-a-user
```

Reply:
This is the only blocker I have - all other feedback I gave, I'm Ok being in a follow-up PR